modules/nf-core/cnvkit/batch/main.nf

```diff
@@ -27,7 +27,8 @@ process CNVKIT_BATCH {
     task.ext.when == null || task.ext.when
 
     script:
-    def args = task.ext.args ?: ''
+    def args = task.ext.args && task.ext.custom_args ? task.ext.args + ' ' + task.ext.custom_args :
+        task.ext.args ?: ''
```

Comment:
Super nice! I was wondering about this the other day actually, whilst explaining to someone that they had to figure out the default args to add something 😅 

I find the logic slightly difficult to follow. Would something like this work and be a bit easier to read? (untested)

```suggestion
    def args = task.ext.args ?: ''
    if(task.ext.custom_args) args += ' ' + task.ext.custom_args
```

Reply:
oh yeah, so much better indeed

Reply:
let me try locally

Reply:
what about that, to keep the same logic for both?
```suggestion
    def args = task.ext.args ?: ''
        args += task.ext.custom_args ? ' ' + task.ext.custom_args : ''
```

Reply:
> Super nice! I was wondering about this the other day actually, whilst explaining to someone that they had to figure out the default args to add something sweat_smile

I had some questions and requests about having this feature, and this morning as I woke up, I had the solution.
And for once, it was a good still a good idea when I finally try it out (not often the case when thinking about things while being half asleep)

Reply:
Honestly, I find the ternary operator harder to read and I'm not sure it's needed here as we don't need to add an empty string if it's not set.

But I don't mind really, the main thing was to keep the logic separate for the two `task.ext` and you've kept that 👍🏻

Reply:
🤯 I can't understand any of that. I general a prefer a lot of easy-to-understand lines as opposed to a few cryptic lines that I can't parse.

Reply:
We have ternary operators everywhere. That's the third in this module alone. And I'm just copying the logic from the previous one